omod/src/main/resources/apps/patient_dashboard_app.json

```diff
@@ -247,5 +247,29 @@
       }
     }
   ]
+},{
```

Comment:
Thanks for working on this, On this line 250 i would do some identification

Reply:
@sherrif10  kindly be more elaborate about your comment.

Reply:
Thanks @slubwama I meant to separate line 250 and add open curry brace no next line

Reply:
@sherrif10 this is noted, thanks.

Reply:
@sherrif10, this requested has been duly effected.

Reply:
@semujju  thanks  LGTM  though am seeing many deletions of config json files cc @slubwama can you confirm  thanks